libsawtooth/src/receipt/store/mod.rs

```diff
@@ -19,7 +19,10 @@
 
 mod error;
 
+use std::convert::TryInto;
```

Comment:
I don't see this used in this commit

Reply:
Line 130 of this file in `impl FromBytes for u64`